eclipse-theia/theia
# Load script in body

#### What it does

Fixes #10927 by loading the scripts in the `body` element rather than the `head`. This ensures that the `body` exists to be accessed when the JS runs.


#### How to test

1. Remove `--mode development` from the `bundle` step in the `examples/browser` and `examples/electron` scripts.
2. Build the application.
3. Run the application in browser and Electron modes.
4. It should load and work normally.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
